src/game.h

```diff
@@ -1094,6 +1094,7 @@ class game
         weather_manager weather; // NOLINT(cata-serialize)
 
     public:
+        std::string weather_eternal = "normal"; // NOLINT(cata-serialize)
```

Comment:
Why are you storing a copy of the option here? It is only read in `weather_manager::update_weather()`, where you could read the option value directly.

You would not need to synchronize it with the option value and changing the option ingame might actually work.

Reply:
It's also used for json serialization and deserialization. I'm not sure where I should write json data to if I remove `weather_eternal`.

Reply:
All options are automatically stored anyway.